app/javascript/App/components/CoinShow/index.js

```diff
@@ -83,145 +65,27 @@ class CoinShow extends Component {
       coinObj,
     } = this.props
 
-    let coinsCollection
-    if (this.state.liveCoinArr.length) {
-      coinsCollection = this.state.liveCoinArr
-    } else {
-      coinsCollection = this.props.coins
-    }
-
     const percentChange1h = {
       positive: coinObj.change1h > 0,
       value: coinObj.change1h,
     }
 
-    const currencyMenu = (
-      <Menu onClick={this.changeCurrencyHandler}>
-        <Menu.Item key="USD">USD</Menu.Item>
-        <Menu.Item key="BTC">BTC</Menu.Item>
-      </Menu>
-    )
-
     return (
       <Fragment>
         <Layout style={{ overflow: 'auto' }}>
           <Content>
             <Row>
-              {/* watchlist */}
-              <Col xs={0} sm={0} m={4} l={4} xl={4}>
-                <div
-                  style={{
-                    border: '1px solid rgba(0, 0, 0, 0.12)',
-                    borderTop: 0,
-                    background: '#fff',
-                  }}
-                >
-                  <CoinList
-                    {...this.props}
-                    watchlistHandler={this.watchlistHandler}
-                  />
-                </div>
-              </Col>
-
               {/* Coin List Button */}
-              <Col xs={24} sm={24} m={20} l={20} xl={20}>
+              <Col xs={24} sm={24} m={24} l={24} xl={24}>
                 <SectionHeader>
-                  {' '}
-                  <HideLarge>
-                    <Button
-                      type="primary"
-                      onClick={() =>
-                        this.props.enableUI('coinListDrawer', {
-                          fullScreen: true,
-                        })
-                      }
-                      style={{ marginRight: '1rem' }}
-                    >
-                      <i
-                        className="material-icons"
-                        style={{
-                          marginLeft: -10,
-                          marginRight: 7,
-                          marginTop: 2,
-                        }}
-                      >
-                        list
-                      </i>
-                      <span
-                        style={{
-                          position: 'relative',
-                          top: -7,
-                        }}
-                      >
-                        Coin List
-                      </span>
-                    </Button>
-                  </HideLarge>
                   <SearchCoins {...this.props} coinShow />
                 </SectionHeader>
                 {/* main content */}{' '}
                 <Row>
-                  {/* currency button and watchlist */}
-                  <Col xs={24} s={24} m={6}>
-                    <ButtonWrap>
```

Comment:
@hanchang if you add this `<ButtonWrap>` component back in it might position the watch button in the top right